model-engine/model_engine_server/entrypoints/start_fastapi_server.py

```diff
@@ -8,7 +8,9 @@
 from typing import List
 
 
-def start_gunicorn_server(port: int, num_workers: int, debug: bool) -> None:
+def start_gunicorn_server(
```

Comment:
wait this is the gateway right? I don't think we need to change things for the worker?